### PR TITLE
Updated prowler-codebuild-role name for CFN StackSets name length limit @varunirv

### PR DESCRIPTION
**Pull request by varunirv**
This change is to fix the issue as reported at
https://github.com/toniblyx/prowler/issues/845

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by w0rmr1d3r — APPROVED** [review]
Looks good to me, let's wait for @toniblyx approval :)

**Comment by toniblyx** [comment]
Awesome, thank you!

**Pull request merged**
